src/essentials/Colors/Colors.ts

```diff
@@ -265,6 +284,7 @@ export const SemanticColorsDarkSchema = {
         }
     },
     background: {
+        default: Colors.neutral[900],
         page: {
             default: Colors.blue.primary[900],
```

Comment:
@jahe as agreed with Anna on Friday, we will only introduce a new semantic key: `background-page-elevation-0`.

It should be mapped to the new `neutral-10`. The rest of the semantic names/values should be untouched.

Thank you!

Reply:
Hey @arturmiglio! I applied the changes we agreed on in 86d8c03.